web/pgadmin/browser/server_groups/servers/databases/schemas/collations/static/js/collation.ui.js

```diff
@@ -160,8 +206,12 @@ export default class CollationSchema extends BaseUISchema {
       setError('copy_collation', errmsg);
       return true;
     }
+    if (isEmptyString(state.version)){
```

Comment:
Why is version mandatory? It is optional in docs - https://www.postgresql.org/docs/current/sql-createcollation.html.